content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: hugo
+title: An even better rOpenSci website? Off Hugo!
+date: '2019-01-10'
+topicid:
+authors:
+  - Maëlle Salmon
+categories: technotes
+tags:
+  - hugo
+---
+
+A bit more than one year ago, rOpenSci [launched its new website design](https://twitter.com/rOpenSci/status/925036483383148545), by the designer [Maru Lango](https://www.marulango.com/). Not only did the website appearance change (for the best!), but the underlying framework too. ropensci.org is powered by [Hugo](https://gohugo.io/), like `blogdown`! Over the last few months, we've made the best of this framework, hopefully improving your browsing experience (and trapping you into binge reading). In this note, we'll go over the main developments, as well as give some Hugo tips. 
+
+# Our website and its source
+
+Skip this section if you just want to know what's become better on our website!
+
+Our website is located at [ropensci.org](https://ropensci.org). Its source lives on GitHub in [the ropensci/roweb2 repo](https://github.com/ropensci/roweb2/). The website is deployed on Netlify from the master branch, and each pull request gets a Netlify preview link too, which is quite handy when reviewing new blog posts.
+
+Nearly all the changes mentioned in this post were made to the _theme_ of our website which you can find [at this folder of the GitHub repo](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts). When modifying Hugo templates, [reading Hugo docs](https://gohugo.io/documentation/) is very helpful. When things go wrong (why doesn't this partial know about the author list?!) head to the [debugging template docs](https://gohugo.io/templates/template-debugging/). And if you can't figure out a way out, Hugo has got a [Discourse forum](https://discourse.gohugo.io/) like [all the cool kids these days](https://discuss.ropensci.org/). Needless to say, another useful tip when learning Hugo templating is to start small, with either a minimal theme and tiny website, or small updates, then your skills will improve.
```

Comment:
While on theme, IMO the space between `<h2>` and the following `<p>` looks a bit compressed

Reply:
I've changed level headers, but not sure it looks better now. Headings have been a problem for a while around here. cf #30